### ISSUE TITLE
mixhash in block RPC headers

### ISSUE THREAD

**Issue by RyanLucchese** [energicryptocurrency/energi#155]
Currently RPC calls which return a block header omit the mixhash - they need to include the mixhash. These calls are `getblock`, `getblockheader` and `getblockheaders`, and possibly more.

Example output:
```
$ energi-cli getblockheader 00001712ade867fb3c5df73ba1afc2784930da2ec346a7736f3fb2b467fe6e2d
{
  "hash": "00001712ade867fb3c5df73ba1afc2784930da2ec346a7736f3fb2b467fe6e2d",
  "confirmations": 13552,
  "height": 3000,
  "version": 536870912,
  "merkleroot": "097a8ddbad8626de95b837a7b8abc47698f76cd6390ecb42f8c33e0ed47db81e",
  "time": 1507224667,
  "mediantime": 1507224612,
  "nonce": 492407,
  "bits": "1e19a926",
  "difficulty": 0.0001522244094078499,
  "chainwork": "0000000000000000000000000000000000000000000000000000000008de1391",
  "previousblockhash": "000013d062cb0c7075c89db9e02ae4433313c347ba79da773ba89962b2bad54e",
  "nextblockhash": "000005ac93c31e88bdeb1dae740af49c5847ef10652bcb166c2615dd442272b5"
}
```

Desired output:
mixhash entry included in block header

**Comment by RyanLucchese** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [mixhash in block RPC headers](https://trello.com/c/5KerQX6o/99-mixhash-in-block-rpc-headers)

**Comment by RyanLucchese** [comment]
Fixed by #190
